decidim-core/lib/decidim/core/engine.rb

```diff
@@ -65,7 +65,8 @@ class Engine < ::Rails::Engine
       initializer "decidim.assets" do |app|
         app.config.assets.paths << File.expand_path("../../../app/assets/stylesheets", __dir__)
         app.config.assets.precompile += %w(decidim_core_manifest.js
-                                           decidim/identity_selector_dialog)
+                                           decidim/identity_selector_dialog
+                                           decidim/start_conversation_dialog)
```

Comment:
Shoudn't this files be specified inside `decidim_core_manifest.js` to keep it everything in the same place?